MVCCore.Mediator.Handler/ValidatorHandler/LoginValidatorHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using FluentValidation;
+using FluentValidation.AspNetCore;
+using MediatR;
+using MVCCore.Mediator.Request;
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace MVCCore.Mediator.Handler.ValidatorHandler
+{
+    public class LoginValidatorHandler : AbstractValidator<LoginRequest>
+    {
+         
+        public LoginValidatorHandler()
+        {
+
+            SetRules();
```

Comment:
write the rule in the constructor itself

Reply:
Done